sdk/core/core-auth/test/index.spec.ts

```diff
@@ -5,6 +5,7 @@ import {
   AzureKeyCredential,
   AzureNamedKeyCredential,
   AzureSASCredential,
+  isKeyCredential,
```

Comment:
Just a little curious if isNamedKeyCredential has any relations with isKeyCredential ?

Reply:
yes, it could have this relation but i didn't implement this way. Considering 1) this may not simplify the code a lot; 2) there is no this kind of releation also in their interface definition https://github.com/Azure/azure-sdk-for-js/blob/3ba97cc633616e2e45578ccc1561f7d105f7f71d/sdk/core/core-auth/src/azureNamedKeyCredential.ts#L9.